homework/zadanie4_five_philosophers.cpp

```diff
@@ -13,39 +13,46 @@ class CFork{
 
 public:
     mutable std::mutex mtx_fork_;
+    
+    CFork(){
+    }
+
+    CFork(CFork&&){
+    }
```

Comment:
po co nam takie puste konstruktory?
Jeśli ich nie ma to i tak będą wygenerowane automatycznie. Nie warto ich pisać.

Reply:
dzieki nim mozliwe jest stworzenie tymczasowego obiektu CFork(), bez konstruktora kupiujacgo i i domyslnego wyrzuca bledy, przy tworzeniu obiektu CFokr(), a nie new CFork(), ale dalej pracuje nad tym

Reply:
jeśli są potrzebne to daj im = default.
Bo w tej chwili konstruktor przenoszący działa niepoprawnie, bo nic nie robi zamiast przenosić.